app/commands/iteration/create.rb

```diff
@@ -5,9 +5,10 @@ class Create
     initialize_with :solution, :submission
 
     def call
+      time = Time.now.utc.to_s(:db)
```

Comment:
Nice.

Reply:
`NOW()` doesn't work with the latest version of MySQL on mac for some reason.